metpy/calc/thermo.py

```diff
@@ -1608,7 +1612,12 @@ def _isen_iter(iter_log_p, isentlevs_nd, ka, a, b, pok):
 
     # Change when Python 2.7 no longer supported
     # Pull out keyword arguments
-    tmpk_out = kwargs.pop('tmpk_out', False)
+    temperature_out = kwargs.get('tmpk_out')
+    if temperature_out:
+        warnings.warn('The use of "tmpk_out" has been deprecated in favor of'
+                      '"temperature_out",', metpyDeprecation)
+    if temperature_out is None:
```

Comment:
Should we go ahead and always take what's passed in with `temperature_out`, with something like:
```python
temperature_out = kwargs.pop('temperature_out', temperature_out)
```